### ISSUE TITLE
proto files for XR 7.6 and and 7.7

### ISSUE THREAD

**Issue by ubaumann** [ios-xr/model-driven-telemetry#21]
Is it possible to get the `telemetry.proto` for XR 7.6 and 7.7, or can I get this information from the device or somewhere else?

**Comment by satyacube** [comment]
The proto has not changed for XR 7.6 and 7.7. You can use https://github.com/ios-xr/model-driven-telemetry/blob/master/protos/734/telemetry.proto

**Comment by severindellsperger** [comment]
Thanks for the reply @satyacube 
Unfortunately, we don't know what was in version 7.6 because the latest available proto files are [version 7.4.3](https://github.com/ios-xr/model-driven-telemetry/commit/7a4356e9cab814c8f88d2c6966ef283e090b5c31). We can only observe that there were new fields introduced in the meantime.

In the attached file [protobuf_del_lo10_ipv6.pdf](https://github.com/ios-xr/model-driven-telemetry/files/10115198/protobuf_del_lo10_ipv6.pdf) (a telemetry message of deleting an ipv6 address from int lo10) we can see a decoded message including the type 17, which is not mentioned in the latest available proto files.

It would be really handy to upload at least the proto files of 7.6 so that we can understand what the newly introduced values stand for.

**Comment by ubaumann** [comment]
I totally agree with type 17. It would be nice to have an up-to-date protofile. Can I get the file from the device?
